### PR TITLE
war tweaks and app filter cleanup

### PR DESCRIPTION
**Pull request by czue**
sorry, the introduction of the `analytics` package made it a little hard to split this out. 

should be pretty easy to read :blowfish: by :blowfish: and most of the commits are super small.

https://github.com/dimagi/commcare-hq/commit/75d3d8c2b01f7a894623c0328be1a755d46f4f5e allows us to delete a couch view that runs over all forms! (will PR that separately for reindexing purposes)

@snopoke @benrudolph @dannyroberts

### CODE REVIEW

File: corehq/apps/reports/filters/forms.py [hunk 1]
```diff
@@ -309,15 +310,15 @@ def remote_forms(self):
 
     @property
     @memoized
-    def remote_forms_info(self):
+    def _remote_forms_info(self):
         """
-            Used for placing remote forms into the drilldown_map. Outputs the same structure as application_forms_info.
+            Used for placing remote forms into the drilldown_map. Outputs the same structure as _application_forms_info.
```
Comment (dimagimon) [inline corehq/apps/reports/filters/forms.py]: E501 line too long (120 > 115 characters)

File: corehq/apps/reports/filters/forms.py [hunk 2]
```diff
@@ -550,20 +528,20 @@ def get_filtered_data(self, filter_results):
...
                     app_id = filter_results[-1]['value']
                     try:
-                        for module in self.remote_forms_info[app_id]['modules']:
+                        for module in self._remote_forms_info[app_id]['modules']:
                             all_forms.extend(module['forms'])
                     except KeyError:
                         pass
                 app_id = self._clean_remote_id(app_id)
-                data.extend([{'value': self.remote_forms[self.make_xmlns_app_key(f['xmlns'], app_id)]} for f in all_forms])
+                data.extend([{'value': self._remote_forms[self.make_xmlns_app_key(f['xmlns'], app_id)]} for f in all_forms])
```
Comment (dimagimon) [inline corehq/apps/reports/filters/forms.py]: E501 line too long (124 > 115 characters)

**Comment by snopoke** [comment]
:+1: lots of 'make this private'

**Comment by benrudolph** [comment]
:potable_water:

**Comment by czue** [comment]
actually made http://manage.dimagi.com/default.asp?189007 to delete the view - going to keep focusing on reports stuff

**Comment by dannyroberts** [comment]
:+1:

**Pull request merged**